app/build.gradle

```diff
@@ -15,14 +15,16 @@ android {
         minSdkVersion rootProject.ext.build['minSdkVersion']
         targetSdkVersion rootProject.ext.build['targetSdkVersion']
         versionCode 1
-        versionName "1.0"
+        versionName AppVersion.versionName
 
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
     }
 
     buildTypes {
         release {
             minifyEnabled false
+            shrinkResources false
```

Comment:
Out of curiosity, what does this config do?

Reply:
It's purpose is to make the APK as small as possible e.g. by removing resources that are not being used:
https://developer.android.com/studio/build/shrink-code